components/brave_wallet/browser/swap_service_unittest.cc

```diff
@@ -1450,10 +1459,14 @@ TEST_F(SwapServiceUnitTest, GetLiFiQuoteError) {
   SetErrorInterceptor(error);
 
   base::MockCallback<mojom::SwapService::GetQuoteCallback> callback;
+
+  auto lifi_error = mojom::LiFiError::New();
+  lifi_error->message = "Invalid request";
+
   EXPECT_CALL(callback, Run(EqualsMojo(mojom::SwapQuoteUnionPtr()),
                             EqualsMojo(mojom::SwapFeesPtr()),
                             EqualsMojo(mojom::SwapErrorUnion::NewLifiError(
-                                mojom::LiFiError::New("Invalid request"))),
+                                std::move(lifi_error))),
                             ""));
```

Comment:
inplace construction with expected error code should work 
`mojom::LiFiError::New("Invalid request", mojom::LiFiErrorCode::kSomethingExpected)))`
(also for next test)

Reply:
updated, but the test still fails